src/SamplesApp/SamplesApp.UITests/Windows_UI_Xaml_Controls/TimePickerTests/UnoSamples_Tests.TimePicker.cs

```diff
@@ -141,14 +137,14 @@ public void TimePickerFlyout_HasDataContextTest()
 			var timePickerFlyout = theTimePicker.Child;
 
 			// Open flyout
-			theTimePicker.FastTap();
+			_app.FastTap("HourTextBlock");
 
 			//Assert
 			Assert.IsNotNull(theTimePicker.GetDependencyPropertyValue("DataContext"));
 			Assert.IsNotNull(timePickerFlyout.GetDependencyPropertyValue("DataContext"));
 
 			// Dismiss the flyout
-			_app.TapCoordinates(10, 10);
+			_app.TapCoordinates(30, 30);
```

Comment:
My best guess is that 10, 10 is clicking the status bar, causing the flyout to not be dismissed.

Reply:
You can also try to get the time picker flyout rect and click outside its bounds.